kingsley/block-template-parts/footer.html

```diff
@@ -0,0 +1,5 @@
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"20px";"bottom":"20px"}}}} -->
```

Comment:
This footer looks very similar to the one that ships in Blockbase. Why do we need a different one?

Reply:
I like the smaller padding in this case. Here's how it looks like with Blockbase footer:
![image](https://user-images.githubusercontent.com/4550351/145519114-1e3a63f9-d93e-4b05-946f-65ec01d3c04e.png)

And how it looks like with a custom footer:
![image](https://user-images.githubusercontent.com/4550351/145519212-260ff696-084a-4145-a215-71bfe55c7736.png)

It's not a huge deal though and if you think it's going to save us headache with individual updates down the road, then I can remove it.

Reply:
I think this approach is fine for now. Once we have the concept of site padding in Gutenberg we can switch to that approach: https://github.com/WordPress/gutenberg/issues/35607

Reply:
I'm not sure I see how site padding will allow us to control the gap **above** the footer.

Reply:
I hope there will be a variable we can reuse in other places.